### ISSUE TITLE
SubViewport prevents _input_event virtual method to be reached

### ISSUE THREAD

**Issue by TweedleDim** [godotengine/godot#84360]
### Godot version

v4.1.3.stable.official [f06b6836a] v4.2.beta4.official [93cdacbb0]

### System information

Godot v4.1.3.stable - Windows 10.0.19045 - Vulkan (Compatibility) - NVIDIA GeForce GTX 1060 6GB (NVIDIA; 31.0.15.3623) - Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 Threads)

### Issue description

If you place an Area2D and its CollisionShape inside a SubViewport, the _input_event() won't be reached. This problem also occurs with an instanciated scene as a child of SubViewport.

Changing the mouse_filter on the SubViewportContainer from pass to ignore doesn't change anything.
Changing handle_input_locally on the SubViewport does nothing either.

### Steps to reproduce

SceneTree looks like :
- SubViewportContainer
  - SubViewport
    - Area2D (with a script)
      - CollisionShape2D

Script of Area2D node :
```
func _input_event(_viewport, _event, _shape_idx):
	print("input event from area2d has been reached")
```
Enable Visible Collision Shapes in the Debug menu.
When the mouse cursor is inside the collision shape we should read a message in the output panel, but there's none.

### Minimal reproduction project

[SubViewport_bug_report.zip](https://github.com/godotengine/godot/files/13238772/SubViewport_bug_report.zip)

**Comment by Sauermann** [comment]
Can you please check, if this is a duplicate of #84258

**Comment by TweedleDim** [comment]
Yes it seems very related, at least my problem is included in it.

I noticed that adding this line of code to my example above:
```
func _ready():
	get_viewport().physics_object_picking = true
```
triggered the code inside _input_event() when the mouse hovered over the area2D, but only once, not on every tick as should be the case.

**Comment by AThousandShips** [comment]
Thank you for reporting, closing as a duplicate of https://github.com/godotengine/godot/issues/84258
